spacy/pipeline/_parser_internals/ner.pyx

```diff
@@ -156,8 +172,6 @@ cdef class BiluoPushDown(TransitionSystem):
         cdef attr_t label
         if name == '-' or name == '' or name is None:
             return Transition(clas=0, move=MISSING, label=0, score=0)
-        elif name == '!O':
-            return Transition(clas=0, move=ISNT, label=0, score=0)
         elif '-' in name:
             move_str, label_str = name.split('-', 1)
             # Hacky way to denote 'not this entity'
```

Comment:
The lines below this need to be removed, as `"x"` won't be in `MOVE_NAMES` anymore.

But perhaps we need to throw a warning here, as old code might still be assuming something like "B-!GPE" is meaningful?